docs/content/en/docs/concepts/_index.md

```diff
@@ -1,8 +1,6 @@
 ---
 title: Concepts
 description: Learn about underlying concepts of the keptn lifecycle toolkit.
-icon: concepts
```

Comment:
we should get rid of this in multiple places - but this can be done in a follow-up :D 
![image](https://github.com/keptn/lifecycle-toolkit/assets/992621/726109d3-39c1-425a-9e54-8d6276d281df)

Reply:
i'll fix that in my typo PR that is in draft mode right now